signal.py

```diff
@@ -238,6 +238,7 @@ def get_buffer(identifier, isGroup):
         cb = "buffer_input_group" if isGroup else "buffer_input"
         logger.debug("Creating buffer for identifier %s (%s)", identifier, "group" if isGroup else "contact")
         buffers[identifier] = weechat.buffer_new(identifier, cb, identifier, "buffer_close_cb", identifier)
+        weechat.hook_signal_send("logger_backlog", weechat.WEECHAT_HOOK_SIGNAL_POINTER, buffers[identifier])
```

Comment:
I think you need to put this below where we rename the buffers, otherwise we'll show logs from the wrong file.